flamingpy/codes/graphs/egraph.py

```diff
@@ -195,15 +195,14 @@ def macronize(self, pad_boundary=False, disp=0.1):
         """
         return EGraph(macronize(self, pad_boundary, disp), macronodes=True)
 
-    def draw(self, **kwargs):
+    def draw(self, backend="matplotlib", **kwargs):
         """Draw the graph state with Matplotlib.
 
         See flamingpy.utils.viz.draw_EGraph for more details.
         """
         from flamingpy.utils.viz import draw_EGraph
 
-        fig, ax = draw_EGraph(self, **kwargs)
-        return fig, ax
+        return draw_EGraph(self, backend=backend, **kwargs)
```

Comment:
this might be a bit dangerous because the output is either a tuple (in the case of matplotlib) or a single figure object (in the case of plotly). I don't know if there is a better solution here.

Reply:
This is for the end-user to be able to modify or personalize the output of the plots. This is an issue only for the syndrome plots that take the axes and draw the syndrome on top of the previous plot .

Reply:
https://github.com/XanaduAI/flamingpy/pull/103#issuecomment-1219572664